app/views/projects/members/index.html.erb

```diff
@@ -118,3 +119,36 @@
     </div>
   </div>
 </div>
+
+
+<%= turbo_frame_tag("members_pagination") do %>
```

Comment:
**nitpick (blocking):** Pagination is not working due to the pagination links being in a `turbo_frame_tag`.

Reply:
Yup I had forgot to push up those changes before you had started reviewing. Updated in 4dd0711fcb61d06e5fe17958f4ec1e787593e564